### ISSUE TITLE
Is there a way to load LocalStorage?

### ISSUE THREAD

**Issue by eguitarz** [garris/BackstopJS#653]
Our app is storing server token in local storage, is there a way to load data from local storage in different sessions?

**Comment by garris** [comment]
I haven't done it -- but I don't see why you couldn't.  Try using the regular localStorage api in an onReady script.

**Comment by eguitarz** [comment]
Thanks! But seems every scenario starts with a fresh environment, all local storage will be cleared out? If that's true, then I'll have to log in with token on every scenario, which is a bit slow and not ideal.

Is there a way not to clear out the localStorage once a scenario is done? (or pass to the next test)

**Comment by garris** [comment]
Yes.  Check the documentation.  You can create a global onReady / onBefore that runs on every test.   Save your state to a file then import that and restore your state using the onBefore/onReady scripts.

**Comment by anton-kulagin** [comment]
And what the logic will be if I create global onReady script and another script to onReady of scenario. Did they both will be applied?

**Comment by eguitarz** [comment]
@anton-kulagin Nope. According to the doc (https://github.com/garris/BackstopJS#running-custom-scripts) If you defined local onReady/onBefore script, backstop will run them instead the global ones.

**Comment by yxb941006** [comment]
sorry, my english is ugly.
I recently had this problem, when i use the regular localStorage api in an onReady script, it will tip me "localStorage is not defined".
![issue](https://user-images.githubusercontent.com/24400950/45354463-42a9c180-b5f0-11e8-9881-d54c1765bb51.png)

**Comment by andrepm06** [comment]
I'm facing the same problem but with sessionStorage.

**Comment by schoenwaldnils** [comment]
With this method I managed to set also the localStorage:
https://github.com/garris/BackstopJS/issues/954#issuecomment-453510935
